homeassistant/components/modbus/__init__.py

```diff
@@ -66,6 +66,7 @@
     CONF_PARITY,
```

Comment:
Lets start from the top with your PR text. You have selected 2 items in "Type of change" you are only allowed to select 1 and this is a new feature, not a quality improvement of the existing code.

The "Checklist" needs to be marked in all fields (and of course you need to have taken the actions).

Documentation is needed for this feature, but that can be added later (before approval).